tests/features/webtools.feature

```diff
@@ -0,0 +1,44 @@
+@api
+Feature: Webtools as administrator
+  In order to manage Webtools on the website
+  As an administrator
+  I want to be able to create, edit and delete block Webtools
+
+  Background:
+    Given these modules are enabled
+      | modules             |
+      | nexteuropa_webtools |
+    And a valid Smartload Url has been configured
+    And I am logged in as a user with the 'administrator' role
+
+  @api
+  Scenario: Create and delete a block 'Map'
```

Comment:
This two cases are very simple and I think that it's not necessary to separate them.